internal/ledger/store/db.go

```diff
@@ -40,7 +40,7 @@ type Iterator interface {
 	// Value returns value itself (ex: record, drop, blob, etc).
 	// Warning: Value is only valid as long as item is valid (until iterator.Next() called), or transaction is valid.
 	// If you need to use it outside its validity, please copy the value.
-	Value() ([]byte, error)
```

Comment:
We mustn't remove the result error here. It's a ok situation in go to return err, when iterator is in the an end